aion/lib/aion/ranking.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Aion.Ranking do
+  @moduledoc """
+  This module provides logic for generating rankings
+  """
+  import Ecto.Query, only: [from: 2]
+  alias Aion.{Repo, User, UserCategoryScore}
+
+  @type user_general_score :: %{user_name: String.t, score: integer}
```

Comment:
This could also be multiline (doing multiline imports etc. makes it clearer in git diffs, because you can see which parts were modified exactly)

Reply:
I am not convinced. I'd prefer to keep it in a single line.

Reply:
Yeah if you want (or if you have an argument why this is better) you can leave it :)

Reply:
It's much more readable to me. Splitting such a small structure into distinct lines seems unnatural.